### PR TITLE
lighthouse update

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
- fixed atmos lockers not having hardsuits
- cryo pipes no longer stack, only 1 pod now as a result

### CODE REVIEW

File: Resources/Maps/lighthouse.yml [hunk 1]
```diff
@@ -71008,10 +71037,14 @@ entities:
   - uid: 15295
     components:
     - type: Transform
+      anchored: False
```
Comment (IamVelcroboy) [inline Resources/Maps/lighthouse.yml]: This one looks funny (see also: Physics)

File: Resources/Maps/lighthouse.yml [hunk 2]
```diff
@@ -78929,11 +78954,15 @@ entities:
   - uid: 122
     components:
     - type: Transform
+      anchored: False
```
Comment (IamVelcroboy) [inline Resources/Maps/lighthouse.yml]: And here

File: Resources/Maps/lighthouse.yml (hunk 1 above)
Comment (deltanedas) [reply Resources/Maps/lighthouse.yml]: yep i removed all the duplicate pipes

**Comment by therealDLondon** [comment]
Finally, atmosians will no longer get trolled

**Review by IamVelcroboy — APPROVED** [review]

**Pull request merged**